vendors/nuvoton/boards/numaker_iot_m487_wifi/aws_demos/application_code/main.c

```diff
@@ -453,7 +454,7 @@ void vAssertCalled( const char * pcFile,
 
 void vApplicationIdleHook( void )
 {
-    const uint32_t ulMSToSleep = 1;
+//    const uint32_t ulMSToSleep = 1;
```

Comment:
Hi is this PR still a work in progress?

Reply:
@sarenameas ,
Yes, it's till in progress and in waiting the approval of reviewers.
The last added 3 commits could resolve  warnings relative to this PR.

Reply:
Could you delete the commented out code?

Reply:
Yes, it's done in commit in eefce60 .